.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Release CLI
+
+on:
+  create:
+    tags:
+      - 'v*'
+
+jobs:
```

Comment:
I haven't tested this but will try a release in a bit and see if it works, may have a followup PR